source/fuzz/fuzzer_util.h

```diff
@@ -252,6 +252,11 @@ void AddLocalVariable(opt::IRContext* context, uint32_t result_id,
                       uint32_t type_id, uint32_t function_id,
                       uint32_t initializer_id);
 
+// Returns OpFunctionParameter instruction corresponding to the function
+// with result id |function_id|.
```

Comment:
The comment suggests that this just returns a single instruction, when really it returns *all* such instructions - please update.

Reply:
Yep, forgot an `s`. Fixed in d69e23d.